vizro-core/docs/pages/discussion/why_vizro.md

```diff
@@ -0,0 +1,9 @@
+# Why Vizro
+
+In the context of comparison with other visualization packages or business intelligence tools, Vizro combines benefits of the speed and user-friendliness of low-code visual analytics tools with the flexibility and power of high-code tools - whilst providing inbuilt visual design, application architecture and coding standardization which also facilitates scaling and reusability.
+
+The simple and largely declarative configuration language follows a “grammar of dashboards” which makes the creation of complex dashboards more intuitive. This leverages modular components, empowering users to rapidly self-serve the assembly of customized dashboards with inbuilt visual styling in minutes, and reduces the amount of “glue code” required, whilst still retaining the ability to enhance functionality almost infinitely through customization and extensions, with standardization increasing the consistency across outputs.
+
+Using configuration rather than a GUI to define the output facilitates collaboration during the creation process, and increases reusability and scalability in a Python setting.
+
+The largely tech-agnostic configuration driven assembly enables utilizing a range of tools, and an inbuilt validation layer provides meaningful feedback to users about configuration choices, thereby increasing flexibility and ease of use.
```

Comment:
Shall we add the image with the config on the left and the dashboard on the right here as well? Seems like this is helpful to understand why vizro as well

Reply:
yes, that could be useful between the 2nd and 3rd paragraphs (until the next explanatory content update)